Assets/Scripts/Player/Body/DamageThresholds.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using Enums;
+
+namespace Player.Body
+{
+    //TODO: should be moved to some sort of config
+    /// <summary>
+    /// Static class to store minimum damage values before an effect can take place on a bodypart.
+    /// EG: an arm needs to be damaged by at least 10 points, before it will start to bleed.
+    /// This is just the minimal value. Once it is reached, there is still a chance the status won't happen right away.
+    /// </summary>
+    public static class DamageThresholds
+    {
+        public static readonly Dictionary<BodyPartStatuses, float> OrganicMinimumRequiredDamage = new Dictionary<BodyPartStatuses, float>
+        {
+            {BodyPartStatuses.Numb, 2f},
+            {BodyPartStatuses.Bruised, 5f},
+            {BodyPartStatuses.Bleeding, 10f},
+            {BodyPartStatuses.Burned, 10f},
+            {BodyPartStatuses.Blistered, 25f},
+            {BodyPartStatuses.Crippled, 50f},
+            {BodyPartStatuses.Severed, 90f}
```

Comment:
These values should be stored somewhere more accessible like a scriptable object. I know it says so in the TODO, but we might as well do it now

Reply:
Dictionaries don't serialize in Unity :( Would not be possible to add them through inspector anyway, so ScriptableObject is just extra overhead.

I could do a List, but then I'd have to filter it each time, instead of being able to directly access by key.

Reply:
That is a good point. I'll resolve this until we have a solution. In my other projects I usually have lists of structs, but I do see the argument for not having overhead.